philter_lite/coordinate_map.py

```diff
@@ -23,14 +25,13 @@ def __init__(self, pattern={"title": "untitled"}, debug=False):
 
         self.map = {}
         self.coord2pattern = {}
-        self.pattern = pattern
-        self.debug = debug
```

Comment:
why were these removed?

Reply:
those are never set and unused

Reply:
doesn't that depend on whether code calls them?

Reply:
I have checked, and it was not used

Reply:
Not used in our other consuming code, but other consuming code might reference those params/attributes (unknown to us).  So, this is a breaking change - now reflected in the new version number (0.5.0). All good.